mlflow/tensorflow/__init__.py

```diff
@@ -515,7 +516,12 @@ def predict(self, data):
                 val = data[df_col_name]
                 if isinstance(val, pandas.DataFrame):
                     val = val.values
+
+                val = val.to_list()
+                if isinstance(val[0], (list, np.ndarray)):
+                    val = np.array(val)
```

Comment:
Why do we only convert to a numpy array if it's a 2D+ array? Could we unconditionally call `np.array()`?